packages/Interface/src/dtk.i

```diff
@@ -0,0 +1,6 @@
+%module DataTransferKit
+
+%{
+#include "DTK_C_API.h"
+%}
+%include "DTK_C_API.h"
```

Comment:
Different includes, one goes to the wrapper cpp file, one is used by SWIG to generated stuff. This is the way SWIG works.